test/nut/sandboxCreate.nut.ts

```diff
@@ -85,14 +85,15 @@ describe('Sandbox Create', () => {
     const sbxName = 'createSbx1';
     const sbxLicenseType = 'Developer';
     const sbxProcess = getSandboxProcess({ SandboxName: sbxName });
+    const expectedCmdResponse = Object.assign({}, sbxProcess, { SandboxUsername: `${hubOrgUsername}.${sbxName}` });
```

Comment:
NIT: `object.assign` again, but you could also call `getSandboxUsername()` too 🤷‍♂️